subsys/shell/Kconfig

```diff
@@ -251,6 +251,14 @@ config SHELL_AUTOSTART
 	help
 	  If enabled, shell will be automatically started.
 
+config SHELL_CMDS_RETURN_VALUE
+	bool "Command to retrieve return value of most recent command"
+	depends on SHELL_CMDS
+	default y
+	help
+		Enables a command to retrieve the return value from the previously executed command.
```

Comment:
Please can you improve this help? What is the command that it enable? What is the range of a return value?

Reply:
I think 'return code' is a better name than 'return value'. This is special, in that it relates to command execution, not just returning a result from a function.

Reply:
I asked before about the name of the command but received no feedback. If changing it is best, I'd be happy to, but I would appreciate some more input from others whether this is the preferred name.

Reply:
Personally, I don't have strong opinion about this. However, what i see is that it would be good to combine shell commands into one root command e.g. `shell` so we have `shell retval` or `shell retcode`. It's not a subject of this PR, though but something that we should consider soon.